concent_api/core/tests/test_unit_model.py

```diff
@@ -196,106 +200,118 @@ def store_report_computed_task_as_subtask():
 class TestDepositAccountValidation(ConcentIntegrationTestCase):
     def setUp(self):
         super().setUp()
-        self.task_to_compute = self._get_deserialized_task_to_compute()
-        self.payer_ethereum_address = self.task_to_compute.requestor_ethereum_address
-        self.payee_ethereum_address = self.task_to_compute.provider_ethereum_address
+        task_to_compute = self._get_deserialized_task_to_compute()
+        self.payer_ethereum_address = task_to_compute.requestor_ethereum_address
 
-        self.client = Client(public_key_bytes=self.PROVIDER_PUBLIC_KEY)
+        self.client = Client(public_key_bytes=self.REQUESTOR_PUBLIC_KEY)
         self.client.clean()
         self.client.save()
 
-        self.deposit_account = DepositAccount()
-        self.deposit_account.client = self.client
-        self.deposit_account.ethereum_address = self.payer_ethereum_address
-
     def test_that_exception_is_raised_when_ethereum_address_has_wrong_length(self):
-        self.deposit_account.ethereum_address = self.payer_ethereum_address + '1'
+        with pytest.raises(ValidationError):
+            deposit_account = DepositAccount(
+                client=self.client,
+                ethereum_address=self.payer_ethereum_address + '1'
+            )
+            deposit_account.clean()
 
+    def test_that_exception_is_raised_when_ethereum_address_has_wrong_type(self):
         with pytest.raises(ValidationError):
```

Comment:
Is it possible to check message added to `ValidationError`? For now it is only one validation and one message, but some others validations may be added and it will be good to check which one raised the `ValidationError`